StockCount/Utils/RefersionKeys.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  RefersionKeys.swift
+//  StockCount
+//
+//  Created by gold on 2019/4/15.
+//  Copyright © 2019 Simple Design Inc. All rights reserved.
+//
+
+import Foundation
+
+let ChinaLink = "https://www.amazon.cn/dp/B01N8SQ8MU/ref=sr_1_7?__mk_zh_CN=%E4%BA%9A%E9%A9%AC%E9%80%8A%E7%BD%91%E7%AB%99&keywords=socket+mobile&qid=1555606674&s=gateway&sr=8-7"
+let EuroCodes = ["AL", "AD", "AM", "AT", "BY", "BE", "BA", "BG", "CH", "CY", "CZ", "DE", "DK", "EE", "ES", "FO", "FI", "FR", "GE", "GI", "GR", "HU", "HR", "IE", "IS", "IT", "LT", "LU", "LV", "MC", "MK", "MT", "NO", "NL", "PO", "PT"]
+let DefaultKey = (pub_key: "pub_9a42760dc57269d2f616", sec_key: "sec_86be30771c1af32a57df", aff_code: "7daad6")
+let Key = [
+    "JP"        : (pub_key: "pub_0fa9f9c60fb103ec2c71", sec_key: "sec_e11ef85cb83bc2b24d31", aff_code: "5413c4"),
+    "AU"        : (pub_key: "pub_9c09ee1b0154d56d2cdd", sec_key: "sec_68c0cf8ad5cc3059befe", aff_code: "b922b8"),
+    "GB"        : (pub_key: "pub_d76b55562cc316faec74", sec_key: "sec_8ef31932ec22f34bc943", aff_code: "5462fa"),
+    "EMEA"      : (pub_key: "pub_81784e9af147411559cb", sec_key: "sec_a367fe786e0811230d5a", aff_code: "4fc514")
```

Comment:
I thought we were going to remove the security keys because we should not expose any secret in the source control?

Reply:
removed sec key from code and added template plist file.